app/cypress.json

```diff
@@ -19,5 +19,6 @@
     "*the-ozone-project.com",
     "*openx.net"
   ],
+  "fixturesFolder": false,
```

Comment:
What does this do?

Reply:
It should stop cypress from creating a fixtures directory inside the cypress directory as our fixtures now live outside of cypress